docs/eBpfExtensions.md

```diff
@@ -73,7 +73,7 @@ functions that override the global helper functions provided by the eBPF runtime
 structure from provided data and context buffers.
 * `context_destroy`: Pointer to `ebpf_program_context_destroy_t` function that destroys a program type specific
 context structure and populates the returned data and context buffers.
-* `required_irql`: IRQL that the eBPF program runs at.
+* `required_irql`: IRQL that the eBPF program runs at when being invoked via bpf_prog_test_run_opts.
```

Comment:
```suggestion
* `required_irql`: IRQL at which the eBPF program is invoked by bpf_prog_test_run_opts.
```

Reply:
So does this mean the IRQL used by bpf_prog_test_run_opts, and the IRQL from a real invocation can be different? If so, the label "required_irql' seems wrong to me.

Reply:
Agreed. Longer term there is a need to have the bpf_prog_test_run_opts specify what IRQL to run the BPF program at and for the extension to be able to validate this, which will be addressed by https://github.com/microsoft/ebpf-for-windows/issues/3157, but that is a larger change. The goal of this is to unblock the BPF performance testing so we can detect regressions.

Before #3157 can be implemented, there needs to be a decision on:
1) Do we need to update the version of the interface for pre-release eBPF core or is a breaking changer acceptable.
2) How can this be passed this via the libbpf API in a way that maintains compatability.

Reply:
Agreed that the label is wrong?  If so, why not change it now?

Reply:
Ok, I take it changing the label would be a breaking change, yes?